### PR TITLE
[dom daint] Update 7.0.UP02-20.11-daint-gpu

### PR DESCRIPTION
**Pull request by twrobinson**

### CODE REVIEW

File: jenkins-builds/7.0.UP02-20.11-dom-gpu [hunk 1]
```diff
@@ -28,6 +28,7 @@
  IDL-8.5.1.eb                                           --set-default-module
  IDL-8.7.2-CSCS.eb
  ipcmagic-0.1-CrayGNU-20.11.eb                          --set-default-module
+ ipcmagic-1.0.1-CrayGNU-20.11.eb
```
Comment (lucamar) [inline jenkins-builds/7.0.UP02-20.11-dom-gpu:31]:
```suggestion
 ipcmagic-1.0.1-CrayGNU-20.11.eb                       --set-default-module
```

File: jenkins-builds/7.0.UP02-20.11-dom-mc [hunk 2]
```diff
@@ -27,6 +27,7 @@
  IDL-8.5.1.eb                                           --set-default-module
  IDL-8.7.2-CSCS.eb
  ipcmagic-0.1-CrayGNU-20.11.eb                          --set-default-module
+ ipcmagic-1.0.1-CrayGNU-20.11.eb
```
Comment (lucamar) [inline jenkins-builds/7.0.UP02-20.11-dom-mc:30]:
```suggestion
 ipcmagic-1.0.1-CrayGNU-20.11.eb                       --set-default-module
```

File: jenkins-builds/7.0.UP02-20.11-dom-gpu (hunk 1 above)
Comment (lucamar) [reply jenkins-builds/7.0.UP02-20.11-dom-gpu:31]: Thanks @twrobinson: I suggest to remove the old release on Dom, so we can move forward with `UpdateEB`.

File: jenkins-builds/7.0.UP02-20.11-dom-mc (hunk 2 above)
Comment (lucamar) [reply jenkins-builds/7.0.UP02-20.11-dom-mc:30]: Thanks @twrobinson: I suggest to remove the old release on Dom, so we can move forward with `UpdateEB`.

**Review by lucamar — CHANGES_REQUESTED** [review]
Thanks @twrobinson: I suggest to remove the old release on Dom, so we can move forward with `UpdateEB`.

**Review by twrobinson — COMMENTED** [review]
Good idea Luca

**Review by lucamar — APPROVED** [review]
Thanks @twrobinson

**Pull request merged**